python/lance/tests/test_types.py

```diff
@@ -218,3 +218,10 @@ def test_pickle(tmp_path):
         pickle.dump(img, fh)
     with (tmp_path / "image").open("rb") as fh:
         assert img == pickle.load(fh)
+
+    img = Image.create(bytearray(b"bytes"))
```

Comment:
is the change to support both bytes and bytearray?

Reply:
yeah, previously we only supported bytes but duckdb reads it out as bytearray so this is just a minor convenience